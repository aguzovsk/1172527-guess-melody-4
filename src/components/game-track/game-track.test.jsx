import React from 'react';
import GameTrack from './game-track.jsx';
import renderer from 'react-test-renderer';

it(`Game track snapshot test.`, () => {
  const tree = renderer.create(<GameTrack
    src="https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg"
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
