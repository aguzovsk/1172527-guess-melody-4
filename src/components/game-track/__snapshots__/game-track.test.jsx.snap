// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game track snapshot test. 1`] = `
<div
  className="game__track"
>
  <div
    className="track"
  >
    <button
      className="track__button track__button--play"
      type="button"
    />
    <div
      className="track__status"
    >
      <audio
        src="https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg"
      />
    </div>
  </div>
</div>
`;
